### PR TITLE
operators

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/5.operators/6.set.md [hunk 1]
```diff
@@ -58,77 +48,43 @@ nebula> GO FROM "player102" OVER follow \
...
-```ngql
-nebula> GO FROM "player102" OVER follow YIELD follow._dst AS id, follow.degree AS Degree, $$.player.age AS Age \
-    UNION /* DISTINCT */ \
-    GO FROM "player100" OVER follow YIELD follow._dst AS id, follow.degree AS Degree, $$.player.age AS Age;
-```
-
-returns the follows:
-
-```ngql
+# UNION can also work with the YIELD statement. The DISTINCT will check duplication by all the columns for every line, and remove duplicated lines if every column is the same.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/5.operators/6.set.md:52]:
```suggestion
# UNION can also work with the YIELD statement. The DISTINCT keyword will check duplication by all the columns for every line, and remove duplicated lines if every column is the same.
```

File: docs-2.0/3.ngql-guide/5.operators/6.set.md [hunk 2]
```diff
@@ -137,51 +93,39 @@ Empty set (time spent 5194/6264 us)
 <left> MINUS <right>
 ```
 
-Operator `MINUS` returns the subtraction (or difference) of two sets A and B (denoted by A - B). Always pay attention to the order of the `<left>` and `<right>`. The set A - B consists of elements that are in A but not in B.
+Operator `MINUS` returns the subtraction (or difference) of two sets A and B (denoted by `A-B`). Always pay attention to the order of the `left` and `right`. The set `A-B` consists of elements that are in A but not in B.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/5.operators/6.set.md:96]:
```suggestion
Operator `MINUS` returns the subtraction (or difference) of two sets A and B (denoted by `A-B`). Always pay attention to the order of `left` and `right`. The set `A-B` consists of elements that are in A but not in B.
```

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/3.ngql-guide/5.operators/4.pipe.md [hunk 3]
```diff
@@ -28,3 +31,23 @@ GO FROM $-.dstid OVER follow;
...
+
+In Nebula Graph {{ nebula.release }}, pipes will affect the performance. Take `A | B` as an example, the effects are as follows:
+
+1. Pipe operators operate synchronously. That is, the data can enter the pipe clause as a whole after the execution of clause `A` before the pipe operator is completed.
+
+2. Pipe operators need to be serialized and deserialized, which is executed in a single thread.
+
+3. If `A` sends a large amount of data to `|`, the entire query request may be very slow. You can try to split this statement.
+
+  1. Send `A` on the application,
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/5.operators/4.pipe.md:45]: Send `A` from the application,

File: docs-2.0/3.ngql-guide/5.operators/4.pipe.md [hunk 4]
```diff
@@ -28,3 +31,23 @@ GO FROM $-.dstid OVER follow;
...
+
+  2. Split the return results on the application,
+
+  3. Send multiple graphd processes concurrently,
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/5.operators/4.pipe.md:49]: Send to multiple graphd processes concurrently,

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**